MSAL/test/app/ios/MSALTestAppCacheViewController.m

```diff
@@ -234,8 +228,8 @@ - (void)viewDidLoad {
     }];
     
     _tokenRowActions = @[ deleteTokenAction, expireTokenAction ];
-    _rtRowActions = @[ tombstoneTokenAction, invalidateAction ];
-    _clientIdRowActions = @[ deleteAllAction ];
+    _rtRowActions = @[ deleteTokenAction, invalidateAction ];
+    _environmentRowActions = @[ deleteAllAction ];
     
     [[NSNotificationCenter defaultCenter] addObserverForName:MSALTestAppCacheChangeNotification
```

Comment:
Yeah, i agree it's a best practice. But in here, it is one of the viewcontroller for the tabbar that will not be unloaded for all intended purposes.